include/crm/common/options.h

```diff
@@ -37,6 +37,7 @@ extern     "C" {
 #define PCMK_OPT_ENABLE_ACL                     "enable-acl"
 #define PCMK_OPT_ENABLE_STARTUP_PROBES          "enable-startup-probes"
 #define PCMK_OPT_FENCE_REACTION                 "fence-reaction"
+#define PCMK_OPT_FENCE_REMOTE_WITHOUT_QUORUM    "fence-remote-without-quorum"
```

Comment:
It'd be reasonable to make this an internal constant in `options_internal.h`. It would be the only internal `PCMK__OPT`, so I'm not tied to that.

Reply:
My thinking here is just that we may want to (a) keep this as hidden as possible and (b) remove it easily in the future